tsconfig.dev.json

```diff
@@ -11,9 +11,14 @@
     "noUnusedLocals": false,
     "noUnusedParameters": false,
     "outDir": "dist",
-    "baseUrl": ".",
+    "baseUrl": "./",
     "paths": {
-      "@clickhouse/client": ["./src/index.ts"]
+      "@clickhouse/client-common": ["packages/client-common/src/index.ts"],
+      "@clickhouse/client-common/*": ["packages/client-common/src/*"],
+      "@clickhouse/client": ["packages/client-node/src/index.ts"],
+      "@clickhouse/client/*": ["packages/client-node/src/*"],
```

Comment:
> exporting public API only from the top level

it is already like that, isn't it?

> use relative paths to import inside a module

it is like that within a particular connector (`import ... from '../utils`), but common imports are done as if it was in a separate library (cause that's exactly how it will be when it's released)